reactioncommerce/reaction
# [WIP] feat: tag admin featured products pagination

RelatedTo: Ticket #4983, PR https://github.com/reactioncommerce/reaction/pull/4980
Impact: **minor**  
Type: **feature**

## Issue

Fix pagination and sorting of `productsByTagId` query

## Solution

Update the `productsByTagId` to properly filter and sorting similarily to the changes to the `catalogItems` query

## Breaking changes

none

## Testing
1. Add a bunch of products, tags, and add tags to those products
1. Open `http://localhost:3000/operator/tags` and select a tag that has a decent number of products
1. Set the order of some of the products and save
1. Ensure that the products that have a priority are in sequential order, and the rest are below them.
1. Ensure that pagination works forwards an backward. This will require you to have more than 10 products added to that tag.
